app_flutter/lib/service/appengine_cocoon.dart

```diff
@@ -115,6 +115,28 @@ class AppEngineCocoonService implements CocoonService {
     }
   }
 
+  @override
+  Future<CocoonResponse<List<String>>> fetchFlutterBranches() async {
+    final String getBranchesUrl = _apiEndpoint('/api/public/get-branches');
+
+    /// This endpoint returns JSON {"Branches": List<String>}
+    final http.Response response = await _client.get(getBranchesUrl);
+
+    if (response.statusCode != HttpStatus.ok) {
+      print(response.body);
```

Comment:
IIRC in the past we never integrated a logger service because it was (1) small detail during the EngRes project and (2) Flutter for web didn't necessarily have the plugin support for it. I believe both of these issues have been wiped away so I created https://github.com/flutter/flutter/issues/52697 to track this.